vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/test/grid-connector-sorting.test.ts

```diff
@@ -0,0 +1,132 @@
+import { expect, fixtureSync, nextFrame } from '@open-wc/testing';
+import { init, getBodyCellContent, setRootItems } from './shared.js';
+import type { FlowGrid, Item } from './shared.js';
+import sinon from 'sinon';
+import { GridColumn } from '@vaadin/grid';
+import { GridSorter } from '@vaadin/grid/vaadin-grid-sorter.js';
+
+describe('grid connector - sorting', () => {
+  let grid: FlowGrid;
+  let columns: GridColumn<Item>[];
+  let sorters: GridSorter[];
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="name"></vaadin-grid-column>
+        <vaadin-grid-column path="price"></vaadin-grid-column>
+      </vaadin-grid>
+    `);
+
+    init(grid);
+
+    columns = [...grid.querySelectorAll('vaadin-grid-column')] as GridColumn<Item>[];
+
+    grid.$connector.setHeaderRenderer(columns[0], { content: 'Name', showSorter: true, sorterPath: 'name' });
+    grid.$connector.setHeaderRenderer(columns[1], { content: 'Price', showSorter: true, sorterPath: 'price' });
+
+    setRootItems(grid.$connector, [
+      { key: '0', name: 'Macbook', price: 2500 },
+      { key: '1', name: 'iPad', price: 1000 }
+    ]);
+    await nextFrame();
+
+    sorters = [...grid.querySelectorAll('vaadin-grid-sorter')] as GridSorter[];
```

Comment:
You can avoid casts by adding a type to querySelectorAll: `grid.querySelectorAll<GridSorter>('vaadin-grid-sorter')`

Reply:
Done.

Reply:
Done.